frontend/src/views/pricing-view.js

```diff
@@ -0,0 +1,151 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import '../components/ex-card.js';
+
+/**
+ * `pricing-view`
+ *
+ * PricingView element.
+ *
+ * @customElement
+ * @polymer
+ */
+class PricingView extends PolymerElement {
+
+    static get template() {
+        return html`
+            <style include="common-styles">
+                :host {
+                    display: block;
+                }
+                
+                .wrapper {
+                    margin: 0 auto; /* Horizontally center the layout grid if there is space around it */
+                    max-width: 960px;
+                    padding: 0 var(--lumo-space-m);
+                }
+
+                #header {
+                    padding: var(--lumo-space-wide-l);
+                    margin-bottom: var(--lumo-space-xl);
+                    box-shadow: 0 0 0 1px var(--lumo-contrast-5pct), var(--lumo-box-shadow-xs);
+                }
+                
+                #header h5 {
+                    font-size: 1.4rem;
+                    margin: 0;
+                }
+                
+                @media screen and (max-width: 759px) {
+                    #header {
+                        text-align: center;
+                    }
+                    
+                    #header vaadin-menu-bar {
+                        display: inline-block;
+                        max-width: 100%;
+                    }
+                }
+                
+                @media screen and (min-width: 760px) {
+                    #header {
+                        display: flex;
+                        align-items: baseline;
+                    }
+                    
+                    #header h5 {
+                        flex: 1;
+                    }
+                }
+                
+                .card-list {
+                    display: flex;
+                    margin: var(--lumo-space-xl) calc(var(--lumo-space-m) * -1);
+                    flex-wrap: wrap;
+                }
+                
+                .card-list > ex-card {
+                    flex: 1 0 210px;
+                    margin: var(--lumo-space-m);
+                }
+                
+                ex-card ul,
+                #footer ul {
```

Comment:
Should this and below rules be simplified to `ul` and `li` respectively? I don't see any other lists on the page.

Reply:
I think it's bad practice to have generic ul/li styles which remove list styling. That could cause issues if someone copies the example but also wants to have normal lists (which look like lists) in the same view.

Other option would be to use a class like `ul.nav {}` and `ul.nav li {}` but I preferred having a duplicated selector in CSS instead of adding extra class names in Java.

Is it ok to keep it as is?

Reply:
Sure. Not a blocker, more of a personal opinion. I'm ok to keep it as it is.